src/renderer/components/modals/BackupContents.tsx

```diff
@@ -81,15 +71,9 @@ export const BackupContents = (props: ModalContentsProps) => {
 				/>
 
 				<Title size="m" style={{ paddingTop: 15 }}>Ignore files</Title>
-				<p style={{ marginTop: 7 }}>Add any files(s) you would like to exclude from this backup.</p>
-
-				<TextButton
-					style={{ marginTop: 5 }}
-					className={styles.NoPaddingLeft}
-					onClick={() => onClickEditIgnore(site)}
-				>
-					Edit files to ignore
-				</TextButton>
+				<p style={{ marginTop: 7 }}>Add any files(s) here that you'd like to exclude from this backup:</p>
```

Comment:
This may have already been addressed, but if we're making design changes we should probably loop Tyson in. He might have some input that could affect whether we want to go for the more complex approach and keep the edit button.

Reply:
I've posted at https://wpengine.slack.com/archives/C06J67S6NF6/p1708522847016129 to loop Tyson and Austin in.